src/content/en/fundamentals/security/encrypt-in-transit/enable-https.md

```diff
@@ -320,7 +320,7 @@ When the content and application layers are well-tuned (see
 remaining TLS performance concerns are generally small, relative to the
 overall cost of the application. Additionally, you can reduce and amortize
 those costs. (For great advice on TLS optimization and generally, see
-[High Performance Browser Networking](http://chimera.labs.oreilly.com/books/1230000000545) by Ilya Grigorik.) See also Ivan Ristic's [OpenSSL Cookbook](https://www.feistyduck.com/books/openssl-cookbook/) and [Bulletproof SSL And TLS](https://www.feistyduck.com/books/bulletproof-ssl-and-tls/).
+[High Performance Browser Networking](https://hpbn.co/) by Ilya Grigorik.) See also Ivan Ristic's [OpenSSL Cookbook](https://www.feistyduck.com/books/openssl-cookbook/) and [Bulletproof SSL And TLS](https://www.feistyduck.com/books/bulletproof-ssl-and-tls/).
```

Comment:
Add `{:.external}`?

Reply:
We haven't been consistent about adding them, and they weren't there before. I'll take the discussion about requiring them, and adding a test to an issue.